db/tests/test_datastore.py

```diff
@@ -103,3 +129,24 @@ def test_setBitcoinKey(self):
     def test_getKeyFromEmptyTable(self):
         self.ks.delete_all_keys()
         self.assertEqual(None, self.ks.get_key("guid"))
+
+    def test_follow_unfollow(self):
+        self.fd.follow(self.u)
+        following = self.fd.get_following()
+        self.assertIsNotNone(following)
+
+        self.assertTrue(self.fd.is_following(self.u.guid))
+
+        self.fd.unfollow(self.u.guid)
+        following = self.fd.get_following()
+        self.assertEqual(following, '')
+
```

Comment:
maybe here add another:

`self.assertFalse(self.fd.is_following(self.u.guid))`

to make sure `.is_following` also works when it's not following.